src/charm.py

```diff
@@ -202,7 +206,7 @@ def _on_certificates_relation_broken(self, event: EventBase) -> None:
         self._delete_private_key()
         self._delete_csr()
         self._delete_certificate()
-        self._configure_amf(event)
+        self.unit.status = BlockedStatus("Waiting for certificates relation")
```

Comment:
I think this previous way of doing it was ok, configure_amf already is responsible for deciding the status and should place the charm in blocked status (since the relation is now broken), I'd rather have status management be in 1 place instead of 2.

Reply:
Hahh ok, good to know!